compiler/testData/codegen/bytecodeText/sam/samWrapperForNullInitialization.kt

```diff
@@ -17,10 +16,8 @@ fun test() {
 }
 
 // @TestKt.class:
-// 1 NEW TestKt\$test\$1
 // 1 NEW kotlin/jvm/internal/Ref\$IntRef
 // 2 NEW
-// 1 INVOKESPECIAL TestKt\$test\$1.<init>
```

Comment:
@pyos Same here 
// 1 INVOKESPECIAL TestKt\$[a-z$]*.<init>

Reply:
This line seems redundant, given that calls to <init> are paired with NEW.